sdk/src/main/java/io/radar/sdk/RadarLocationReceiver.kt

```diff
@@ -98,10 +98,11 @@ class RadarLocationReceiver : BroadcastReceiver() {
 
     @SuppressLint("MissingPermission")
     override fun onReceive(context: Context, intent: Intent) {
+        Radar.logger.d("Broadcast Received: ${intent.action}")
         when (intent.action) {
             ACTION_BUBBLE_GEOFENCE, ACTION_SYNCED_GEOFENCES -> {
                 val event = GeofencingEvent.fromIntent(intent)
-                event?.triggeringLocation?.also {
+                event.triggeringLocation.also {
```

Comment:
From the docs it does seem like `event` and `triggeringLocation` could be null:

https://developers.google.com/android/reference/com/google/android/gms/location/GeofencingEvent#fromIntent(android.content.Intent)

Reply:
The latest android API mark these as "recently not null", and there's a lint warning on the unnecessary null checks, which is why I removed them.

Reply:
digging in, this looks like it only returns null if the given intent is null.

Reply:
I guess that means older versions of the API could return null?  I think we need to go by what the docs say, which claims it could be null.

Reading up a little on `@RecentlyNonNull`, it seems like we should disable it in the linter because:
1) It's not clear which version it's ever referring to (recently as of when?)
2) It contradicts the docs, which I think has to be the source of truth we rely on.

Reply:
I have the decompiled source in Android Studio (for version 18.0.0, which is packaged with the sdk). That said, I'll just add the check back for now, given the documentation.

Reply:
done.